pulseapi/entries/views.py

```diff
@@ -431,13 +430,9 @@ def post(self, request, *args, **kwargs):
                     encdata = thumbnail['base64']
                     proxy = ContentFile(base64.b64decode(encdata), name=name)
                     request_data['thumbnail'] = proxy
-            except:
+            except thumbnail.DoesNotExist:
                 pass
 
-            # we need to split out creators, because it's a many-to-many
-            # relation with a Through class, so that needs manual labour:
-            creator_data = request_data.pop('creators', [])
```

Comment:
we definitely have tests that specifically test for this functionality though so you would also need to remove those tests.

Reply:
The test for this exists in the tests.py file? I'm assuming its this function def setup_entries(test, creator_users).

Reply:
these are the tests you'll likely need to modify:
https://github.com/mozilla/network-pulse-api/blob/e83bec051fd17098118e1e901ce60cb4db716d7b/pulseapi/entries/test_views.py#L95-L115

https://github.com/mozilla/network-pulse-api/blob/e83bec051fd17098118e1e901ce60cb4db716d7b/pulseapi/entries/test_views.py#L117-L150

https://github.com/mozilla/network-pulse-api/blob/e83bec051fd17098118e1e901ce60cb4db716d7b/pulseapi/entries/test_views.py#L204-L230

https://github.com/mozilla/network-pulse-api/blob/e83bec051fd17098118e1e901ce60cb4db716d7b/pulseapi/entries/test_views.py#L759-L803

Reply:
I should modify these 4 tests. Not remove them completely right?

Reply:
yes, modify them